code/modules/admin/verbs/cinematic.dm

```diff
@@ -1,21 +1,14 @@
-/client/proc/cinematic(cinematic as anything in list("explosion", null))
-	set name = "cinematic"
-	set category = "Debug"
-	set desc = "Shows a cinematic."	// Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
-	set hidden = 1
+/client/proc/cinematic()
+	set name = "Cinematic"
+	set category = "Event"
+	set desc = "Shows a cinematic." // Intended for testing but I thought it might be nice for events on the rare occasion Feel free to comment it out if it's not wanted.
+	set hidden = TRUE
+
 	if(!SSticker)
 		return
 
-	if(!check_rights(R_MAINTAINER))
+	var/datum/cinematic/choice = input(usr, "Choose a cinematic to play to everyone in the server.", "Choose Cinematic") in sort_list(subtypesof(/datum/cinematic), GLOBAL_PROC_REF(cmp_typepaths_asc))
```

Comment:
You definitely need a `check_rights(R_EVENT)` here.

Also, I don't think you really need to implement `sort_list` just for this purpose; we don't even really need to take a copy of the input list since we're generating it right here for this sole purpose.